src/Maths/Silk.NET.Maths/Simd/Simd64.Not.cs

```diff
@@ -0,0 +1,139 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if INTRINSICS
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+#if SSE
+using System.Runtime.Intrinsics.X86;
+#endif
+#if AdvSIMD
+using System.Runtime.Intrinsics.Arm;
+#endif
+
+namespace Silk.NET.Maths
+{
+    public static unsafe partial class Simd64
+    {
+        /// <summary>
+        /// Performs hardware-accelerated Not on 64-bit vectors.
+        /// </summary>
+        [MethodImpl(Scalar.MaxOpt)]
+        public static Vector64<T> Not<T>(Vector64<T> vector) where T : unmanaged
```

Comment:
Seems like we can make this whole method
```cs
if (Simd64.IsHardwareAccelerated) return Simd64.Xor(vector, Simd64<T>.AllBitsSet);
return Other(vec);
```

Reply:
._.